templates/etc/nginx/sites-available/default.conf.j2

```diff
@@ -177,6 +177,21 @@
 #       Optional. ECC curve enabled for this server. By default, 'secp384r1'.
 #
 #
+#   - item.ssl_verify_client: True/False
+#      Optional. Requests the client certificate and verifies it if the
+#      certificate is present. 
+#
+#
+#   - item.ssl_client_certificate: ''
+#      Optional. Specifies a file with trusted CA certificates in the PEM
+#      format used to verify client certificates.
+#
+#
+#   - item.ssl_crl: ''
+#      Optional. Specifies a file with revoked certificates (CRL) in the PEM
+#      format used to verify client certificates.
```

Comment:
This documentation format is deprecated [as it says on the top](https://github.com/debops/ansible-nginx/blob/master/templates/etc/nginx/sites-available/default.conf.j2#L8-L9). Can you move this to [defaults-detailed.rst](https://github.com/debops/ansible-nginx/blob/master/docs/defaults-detailed.rst#nginx-servers)?